week-1-ts-ecosystem/w01e02-enter-the-world-of-typescript/src/framework/dom-creators.ts

```diff
@@ -1,4 +1,4 @@
-function createElement(tagName = 'div', className = ''): HTMLElement {
+function createElement(tagName = 'div', className = ''): HTMLDivElement | HTMLParagraphElement | HTMLElement {
```

Comment:
jeśli Twój `tagName` będzie typem generycznym - to poprawisz typowanie i wnioskowanie TS